test/manifest.ts

```diff
@@ -375,10 +475,40 @@ describe('Manifest', () => {
       const checkpoint = (msg: string, type: CheckpointType) =>
         logs.push([msg, type]);
       const pr = await new Manifest({github, checkpoint}).pullRequest();
+      mock.verify();
       expect(pr).to.equal(22);
       expect(logs).to.eql([
+        [
+          'Found version 3.2.1 for node/pkg1 in ' +
+            '.release-please-manifest.json at abc123 of main',
+          CheckpointType.Success,
+        ],
+        [
+          'Failed to find version for node/pkg2 in ' +
+            '.release-please-manifest.json at abc123 of main',
+          CheckpointType.Failure,
+        ],
+        [
+          'Found version 1.2.3 for python in ' +
+            '.release-please-manifest.json at abc123 of main',
+          CheckpointType.Success,
+        ],
+        [
+          'Bootstrapping from .release-please-manifest.json at tip of main ' +
+            'for missing paths [node/pkg2]',
+          CheckpointType.Failure,
+        ],
+        [
+          'Failed to find version for node/pkg2 in ' +
+            '.release-please-manifest.json at tip of main',
+          CheckpointType.Failure,
+        ],
         ['Processing package: Node(@node/pkg1)', CheckpointType.Success],
         ['Processing package: Node(@node/pkg2)', CheckpointType.Success],
+        [
+          'Falling back to default version for Node(@node/pkg2): 1.0.0',
+          CheckpointType.Failure,
+        ],
         ['Processing package: Python(foolib)', CheckpointType.Success],
```

Comment:
I feel like this kind of output will be useful to end-users trying to debug why a package is getting the version it is